src/components/auth/Signup.tsx

```diff
@@ -19,22 +19,24 @@ const SignUp: React.FC = () => {
 
   const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault();
-    setError('');
+    setError("");
     try {
       // Create user in Firebase
-      const userCredential: UserCredential = await createUserWithEmailAndPassword(auth, email, password);
+      const userCredential: UserCredential =
+        await createUserWithEmailAndPassword(auth, email, password);
       const user = userCredential.user;
 
       // Obtain ID token
       const idToken = await user.getIdToken();
 
       // Send user data to backend
-      const backendURL = import.meta.env.VITE_API_BASE_URL || "http://localhost:8000";
+      const backendURL =
+        import.meta.env.VITE_API_BASE_URL || "http://localhost:8000";
       const response = await fetch(`${backendURL}/signup`, {
         method: "POST",
         headers: {
           "Content-Type": "application/json",
-          "Authorization": `Bearer ${idToken}`,
+          Authorization: `Bearer ${idToken}`,
```

Comment:
Hello @hono1030 !
Have you tested that removing the quotes from "Authorization" works fine?